packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -53,28 +57,26 @@ export const RootScreen = ({ isReadyToSetupBackend }: RootScreenProps) => {
     () => dispatch(enterBackground())
   )
 
-  useEffect(() => {
-    if (accountStatus === Status.SUCCESS || accountStatus === Status.ERROR) {
-      setIsInittng(false)
-    }
-  }, [accountStatus])
-
   return (
     <>
-      <SplashScreen />
-      {isInitting && !hasAccount ? null : (
+      <SplashScreen
+        canDismiss={
+          accountStatus === Status.SUCCESS || accountStatus === Status.ERROR
+        }
+      />
+      {foundUser ? (
         <Stack.Navigator
           screenOptions={{ gestureEnabled: false, headerShown: false }}
         >
           {updateRequired ? (
             <Stack.Screen name='UpdateStack' component={UpdateRequiredScreen} />
-          ) : !hasAccount ? (
+          ) : !foundUser ? (
```

Comment:
Can it ever get to this branch? `foundUser` is the condition for the outer ternary

Reply:
stupid! undid this change. it wasn't doing much